apps/L1E/ticket-booking/ticket-booking-frontend/src/components/AdminHeader.tsx

```
@@ -0,0 +1,26 @@
+export const AdminHeader = ({ activeTab, setActiveTab }: AdminHeaderProps) => {
+    return (
+        <header className="w-full mt-4 px-6 bg-white">
+            <div>
+                <div className="flex justify-between pl-2">
+                    <div className="flex gap-1 items-center mb-[14px]">
+                        <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20" fill="none">
+                            <circle cx="10" cy="10" r="10" fill="#00B7F4" />
+                        </svg>
+                        <h2 data-cy="logo-text" className="text-2xl font-semibold">TICKET BOOKING</h2>
+                    </div>
+                    <img src={"https://media.gettyimages.com/id/156303396/nl/foto/nerd-student-making-a-funny-smiling-face.jpg?s=2048x2048&w=gi&k=20&c=1cNfJMBwkqDDzcFcdbGW6AYg0waGFQLpFNHD2jDIm4w="} className="w-8 h-8 rounded-full object-cover object-center" />
```

ESLint

[error] Found a string with entropy 4.82 : "https://media.gettyimages.com/id/156303396/nl/foto/nerd-student-making-a-funny-smiling-face.jpg?s=2048x2048&w=gi&k=20&c=1cNfJMBwkqDDzcFcdbGW6AYg0waGFQLpFNHD2jDIm4w="

(no-secrets/no-secrets)